django/urls/conf.py

```diff
@@ -57,6 +57,12 @@ def include(arg, namespace=None):
 def _path(route, view, kwargs=None, name=None, Pattern=None):
     from django.views import View
 
+    if kwargs is not None and not isinstance(kwargs, dict):
```

Comment:
`None` will still fail this check:
```suggestion
    if not isinstance(kwargs, dict):
```

Reply:
Kwargs needs to be different than none because otherwise a TypeError would be raised on every URL path that does not have kwargs.

Reply:
`not` sure makes a huge difference doesn't it. never mind! 🤕

Reply:
I think you are missing the whole point, it needs to be diffent than None and at the same time be different than a dict, try to do what you suggested then run the tests, you'll get me.

Reply:
I got you. When I skimmed the ticket an hour ago there was some shorthand that stuck in my short-term memory ("not a dict"), and that's all I was trying to express about my mistake, not trying to parse your line of code which works just fine. This is why I shouldn't do drive-by reviews on Friday nights (as it is here, in my timezone). Forgive me! 🙏🏻

Reply:
Hahahah, it's fine man, no worries.